### PR TITLE
[7642] Display HESA duplicates in support console

### PR DESCRIPTION
**Pull request by stevehook**
### Context
This PR adds a page to the support console with the list of duplicate trainee groups that we have identified.

Out of scope: Functions to act up the duplicates, e.g. accept, reject and delete etc.

### Changes proposed in this pull request
The page we've added looks like this:
image

### Guidance to review

The list of duplicates just reuses the existing trainee card component that we use for trainee lists generally. They are grouped by a space for now (open to suggestions about UX here).

### Important business

- [x] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [x] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [x] Do we need to send any updates to DQT as part of the work in this PR?
- [x] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: app/views/system_admin/_tab_nav.html.erb [hunk 1]
```diff
@@ -8,6 +8,7 @@
   { name: "Dead jobs", url: dead_jobs_path },
   { name: "Pending TRN", url: pending_trns_path },
   { name: "Pending awards", url: pending_awards_path },
+  { name: "Duplicate HESA trainees", url: duplicate_hesa_trainees_path },
```
Comment (tomtrentham) [inline app/views/system_admin/_tab_nav.html.erb:11]: Do we want to feature flag this?

File: app/views/system_admin/duplicate_hesa_trainees/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<%= render PageTitle::View.new(i18n_key: "duplicate_hesa_trainees.index") %>
+
+<div class="govuk-grid-row">
+  <div class ="govuk-grid-column-full">
+    <%= render "system_admin/tab_nav" %>
+
```
Comment (tomtrentham) [inline app/views/system_admin/duplicate_hesa_trainees/index.html.erb:6]: Can we display a title and have the the total number of duplicate records in parentheses after the header, i.e. Duplicate HESA trainees (25), as per the ticket?

**Review by tomtrentham — APPROVED** [review]
Great stuff

File: app/controllers/system_admin/duplicate_hesa_trainees_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SystemAdmin
+  class DuplicateHesaTraineesController < ApplicationController
+    before_action { require_feature_flag(:duplicate_checking) }
+
+    helper_method :duplicate_trainees_count
+
+    def index
+      @potential_duplicate_trainees = PotentialDuplicateTrainee.grouped .page(params[:page] || 1)
```
Comment (Reyko) [inline app/controllers/system_admin/duplicate_hesa_trainees_controller.rb:10]:
```suggestion
      @potential_duplicate_trainees = PotentialDuplicateTrainee.grouped.page(params[:page] || 1)
```

**Review by Reyko — APPROVED** [review]

**Comment by tomtrentham** [comment]
@stevehook I think we should probably turn on the feature flag for this now.

**Pull request merged**